src/Microsoft.SqlTools.ManagedBatchParser/ReliableConnection/SqlConnectionHelperScripts.cs

```diff
@@ -47,7 +47,9 @@ @rc 			int
 ";
 
     public const string GetOsVersion = @"SELECT OSVersion = RIGHT(@@version, LEN(@@version)- 3 -charindex (' on ', LOWER(@@version)))";
-    public const string GetClusterEndpoints = @"IF OBJECT_ID (N'master.dbo.cluster_endpoint_info') IS NOT NULL
+    public const string GetClusterEndpoints = @"IF OBJECT_ID (N'sys.dm_cluster_endpoints') IS NOT NULL
```

Comment:
So if the DMV doesn't exist then it looks like you're just going to return an empty result set instead of falling back to the old logic since it won't actually error out. If we really want to keep supporting old CTPs (is that actually necessary? Alan said we didn't need to do that for the dashboard) then you'll want to change this to take that into account.

Reply:
Or at least I think it will..not actually sure the behavior off hand of calling .Read() when there isn't a result set. Maybe it'll work as intended?

Reply:
Oh thanks! Will verify this errors as I need that to happen. Reason being, this would immediately break all existing CTP 3.2 customers and we don't want that. I think "support last month's release" is the acceptable limit for us. In my other review I'm deleting older-then-3.2 code support.

Reply:
Verified you were right and updated.